tests/require.rs

```diff
@@ -728,3 +728,55 @@ fn issue_1643_args_mutually_require_each_other() {
 
     app.get_matches_from(&["test", "-u", "hello", "-r", "farewell"]);
 }
+
+#[cfg(debug_assertions)]
+#[test]
+#[should_panic(expected = "Argument or group specified in 'requires*' for 'config' does not exist")]
+fn requires_invalid_arg() {
+    let _ = App::new("prog")
+        .arg(Arg::with_name("config").requires("extra").long("config"))
+        .try_get_matches_from(vec!["", "--config"]);
+}
```

Comment:
Would it be possible to also print the name passed into `requires`? 
```
Argument or group 'extra', specified in 'requires*' for 'config', does not exist
```

Reply:
Commented the reason above